### ISSUE TITLE
nu doesn't handle resizing inteligently

### ISSUE THREAD

**Issue by ashpil** [nushell/reedline#477]

**Comment by fdncred** [comment]
This could be related to https://github.com/nushell/reedline/pull/451

**Comment by rgwood** [comment]
> an experience different than some terminals, and one that I'd argue is worse

I agree – the UX has been much worse for me on Windows Terminal since the change.

**Comment by fdncred** [comment]
@rgwood I'm up for a revert. @sholderbach any thoughts?

**Comment by sholderbach** [comment]
D'accord @fdncred. Yes with the right prompt it is pretty obvious. Don't know how much of an issue the resize on launch is with the old resize logic enabled. There might be more annoying artifacts on resizing manually when we update on the resize event. (e.g. Alacritty seemed to have a higher polling rate for resizes compared to kitty for example leading to screen salad).

Do we want to track down the commit and disect it or reimplement the minimal logic from first principles?

**Comment by fdncred** [comment]
@sholderbach I think the commit is mentioned above. Does it look like the right one? https://github.com/nushell/reedline/pull/451

**Comment by sholderbach** [comment]
Ah seems manageable. Let me quickly bash out a PR

**Comment by fxshlein** [comment]
Hi! Has anything happened on this? I switched to nushell as my main shell a few weeks ago, and I love it, but sadly resizing is pretty much completely broken for me, which is pretty annoying 😄

**Comment by fdncred** [comment]
@fxshlein  Agreed, it's annoying. We've been round and round with this with not a lot of success. I guess we're kind of waiting for someone who is annoyed enough to submit a PR that fixes the issue. Trust me when I say, that we'd love for this to happen. I, for one, am tired of answering questions about it on Discord, LOL.
